src/sql/util.py

```diff
@@ -157,6 +157,84 @@ def show_deprecation_warning():
     )
 
 
+def check_duplicate_arguments(
+    magic_execute, cmd_from, args, allowed_duplicates
+) -> bool:
+    """
+    Raises UsageError when duplicate arguments are passed to magics.
+    Returns true if no duplicates in arguments or aliases.
+    """
+
+    aliased_arguments = {}
+    unaliased_arguments = []
+
+    for decorator in magic_execute.decorators:
+        assert isinstance(decorator.args, tuple)
+        dec_args = decorator.args
+        if len(dec_args) > 1:
+            aliased_arguments[dec_args[0]] = dec_args[1]
+        else:
+            if dec_args[0].startswith("--") or dec_args[0].startswith("-"):
+                unaliased_arguments.append(dec_args[0])
+
+    if len(args) <= 1:
```

Comment:
please add a comment here why we are returning True if `len(args) <=1`. Also, this condition can be the first statement in the function since it doesn't seem to depend on the previous statements

Reply:
Added comments and moved to top [here](https://github.com/ploomber/jupysql/pull/908/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R185-R188)